glitter/blocks/carousel/templates/glitter/blocks/carouselblock_content.html

```diff
@@ -9,7 +9,7 @@
                 {% for image in carousel_images %}
                     <li data-target="#carousel-{{ content_block.id }}" data-slide-to="{{ forloop.counter0 }}" {% if forloop.first %}class="active"{% endif %}></li>
                     <h2>{{ image.title }}</h2>
-                    <p>{{ image.subtitle }}</p>
+                    <p>{{ image.subtitle | linebreaks }}</p>
```

Comment:
https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#linebreaks

Are you sure?